src/ArborX_DistributedTree.hpp

```diff
@@ -33,11 +35,19 @@ namespace ArborX
 template <typename MemorySpace>
 class DistributedTree
 {
+  using BoundingVolume = typename BVH<MemorySpace>::bounding_volume_type;
```

Comment:
What is the intent here?  If it is about making sure that `TopTree` bounding volume is consistent with `BottomTree`, then you should deduce `BoundingVolume` from `BottomTree` (possibly from the return type of `BottomTree::bounds()`.

Reply:
This is more in preparation for `DistributedTree` to be templated on `BoundingVolume`, as I wanted to minimize changes.